packages/salesforcedx-vscode-apex/src/commands/apexTestSuite.ts

```diff
@@ -28,19 +28,20 @@ type ApexTestSuiteOptions = { suitename: string; tests: string[] };
 
 const listApexClassItems = async (): Promise<ApexTestQuickPickItem[]> => {
   const apexClasses = await vscode.workspace.findFiles(`**/*${APEX_CLASS_EXT}`, SFDX_FOLDER);
-  const apexClassItems = apexClasses
-    .filter(apexClass => {
-      const fileContent = readFileSync(apexClass.fsPath).toString();
-      return IS_TEST_REG_EXP.test(fileContent);
-    })
-    .map(apexClass => ({
-      label: basename(apexClass.toString(), APEX_CLASS_EXT),
-      description: apexClass.fsPath,
-      type: TestType.Class
-    }))
-    .sort((a, b) => a.label.localeCompare(b.label));
-
-  return apexClassItems;
+  const apexClassItems = await Promise.all(
+    apexClasses
```

Comment:
duplicate code.
if you solve it in the other one, then share it, it'll be solved for both.